lib/Install.js

```diff
@@ -117,6 +124,129 @@ function install(
   return 'SuccessfullyInstalled';
 }
 
+async function downloadFileNative(url, filePath) {
+  return new Promise((resolve, reject) => {
+    const file = fs.createWriteStream(filePath);
+    let fileInfo = null;
+
+    const request = https.get(url, (response) => {
+      if (response.statusCode !== 200) {
+        fs.unlink(filePath, () => {
+          reject(new Error(`Failed to get '${url}' (${response.statusCode})`));
+        });
+        return;
+      }
+
+      fileInfo = {
+        mime: response.headers['content-type'],
+        size: parseInt(response.headers['content-length'], 10),
+      };
+
+      response.pipe(file);
+    });
+
+    // The destination stream is ended by the time it's called
+    file.on('finish', () => resolve(fileInfo));
+
+    request.on('error', (err) => {
+      fs.unlink(filePath, () => reject(err));
+    });
+
+    file.on('error', (err) => {
+      fs.unlink(filePath, () => reject(err));
+    });
+
+    request.end();
+  });
+}
+
+const PKG = 'elm-explorations/test';
+const VERSION = '1.2.2';
+
+async function installUnstableTestMaster(
+  project /*: typeof Project.Project */
+) {
+  console.log(
+    chalk.yellow(`Using the master version of ${PKG} in place of ${VERSION}.`)
+  );
+  console.log(
+    chalk.yellow(
+      `Note: You will need to rm -rf ~/.elm and ./elm-stuff (or use the \`elm-test uninstall-unstable-test-master\` command) afterwards to get back to the ${VERSION} version.`
```

Comment:
One last question: I’m wondering if people will wonder “Hmm, which approach should I take? Delete two folders, or run the black box command?” Do we want to mention _only_ `elm-test uninstall-unstable-test-master`?

Reply:
Since we're saying what we're doing during these commands (with the `chalk.yellow.dim` stuff), I think we can remove the mention of `rm -rf` here, yes. I'll do that